src/components/mdx/link.js

```diff
@@ -1,6 +1,6 @@
 import { ExternalLinkIcon } from '@chakra-ui/icons'
 import { Link as ChakraLink } from '@chakra-ui/react'
-import NextLink from 'next/link'
+import { Image, Text } from '@chakra-ui/react'
 import React from 'react'
 
```

Comment:
thanks for the review!

this helps me understand why the code was [originally written with `ChakraLink` inside `NextLink`](https://github.com/pangeo-data/pangeo.io/pull/48/files#diff-67b30d162c50d0b059157ecdd36cf898c5bdecab148fe027c6980616c4f03759). I changed it to avoid the hydration warning from nested `<a>`, but will try this `as={NextLink}` approach instead.